source/AI.cpp

```diff
@@ -3292,20 +3292,40 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 			if(shift)
 				ship.SetTargetShip(shared_ptr<Ship>());
 
-			double closest = numeric_limits<double>::infinity();
+			double best = numeric_limits<double>::infinity();
 			bool foundEnemy = false;
 			bool foundAnything = false;
+			bool distancePriority = Preferences::Has("Board target");
```

Comment:
Why have this as a variable? It's only used once.

Reply:
I didn't think to change it but to make it clearer is a good justification of why it is that way, because right now its a bool so how would one guess that "board target" = true means the closest? 
It was changed from board closest or smth.
Maybe I should somehow make it a text or smth? Like to prepare for the 3 options?

Reply:
> how would one guess that "board target" = true means the closest?

We can add a comment saying that the strategy used here is for closest.